File: WWW/DesktopModules/NVCMS.Video/Manager/template/Main.ascx.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;
using DotNetNuke;
using DotNetNuke.Entities;

namespace DesktopModules.TinTuc.Manager.template
{
    public partial class Main : DotNetNuke.Entities.Modules.PortalModuleBase
    {
        protected void Page_Load(object sender, EventArgs e)
        {
            string control = DotNetNuke.Common.Globals.ResolveUrl(this.TemplateSourceDirectory +"/inc_list.ascx");
            DotNetNuke.Entities.Modules.PortalModuleBase module;
            if (Request.QueryString["mod"] != null)
            {                
                switch (Request.QueryString["mod"])
                {
                    case "list":
                        control = DotNetNuke.Common.Globals.ResolveUrl(this.TemplateSourceDirectory + "/inc_list.ascx");
                        break;
                    case "edit":
                        control = DotNetNuke.Common.Globals.ResolveUrl(this.TemplateSourceDirectory + "/inc_edit.ascx");
                        break;
                    default:
                        break;
                        
                }                
            }
            try
            {
                module = this.LoadControl(control) as DotNetNuke.Entities.Modules.PortalModuleBase;
                module.ModuleConfiguration = this.ModuleConfiguration;
                paneControl.Controls.Add(module);                
            }
            catch { }
        }
    }
}

File: WWW/DesktopModules/NVCMS.School/Manager/template/Main.ascx.vb
Imports System
Imports System.Web
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports DotNetNuke
Imports DotNetNuke.Services.Exceptions
Namespace NVCMS.Modules.SchoolWebsite
    Partial Class Maintemplate
        Inherits Entities.Modules.PortalModuleBase
        Protected Sub Page_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load
            Try
                'If Not IsPostBack Then

                Dim DynamicPage As String
                Select Case Request.Item("mod")
                    Case "add", "edit"
                        DynamicPage = DotNetNuke.Common.ResolveUrl(Me.TemplateSourceDirectory & "/inc_edit.ascx")
                    Case Else
                        DynamicPage = DotNetNuke.Common.ResolveUrl(Me.TemplateSourceDirectory & "/inc_list.ascx")
                End Select
                Dim objModule As Entities.Modules.PortalModuleBase = CType(Me.LoadControl(DynamicPage), DotNetNuke.Entities.Modules.PortalModuleBase)
                If Not objModule Is Nothing Then
                    objModule.ModuleConfiguration = Me.ModuleConfiguration
                    phDynamicPlaceHolder.Controls.Add(objModule)
                End If
                'End If
            Catch ex As Exception
                ProcessModuleLoadException(Me, ex)
            End Try
        End Sub
    End Class
End Namespace